### DISCUSSION TITLE
How i can get the distance measurement from the anchors

### DISCUSSION THREAD

**Discussion by hssjung** [bitcraze/discussions#641] [Q&A] [answered]
Hi there. 
I'm a new member here and would like to collect the distance from an anchor during a drone flight.
I've checked for examples but can't find any, so I'm asking for help. 
Thanks.

**Comment by krichardsson** [comment, marked as answer]
I assume you are using the Loco system.
To get the distance to an anchor you must use the TWR mode, the distance to the anchors are available as a log variables in
`ranging.distance0`, `ranging.distance1`, `ranging.distance2` and so on.

If you are using one of the TDoA modes, you can get the **difference** in distance to the anchors in a pair. Set the desired anchor ids of the pair in parameters [`tdoaEngine.logId`](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/params/#tdoaenginelogid) and [`tdoaEngine.logOthrId`](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/params/#tdoaenginelogothrid). The tdoa value will be available in the log variable [`tdoaEngine.tdoa`](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/logs/#tdoaenginetdoa). It is only possible to log one pair.

[Examples of how to get log data](https://github.com/bitcraze/crazyflie-lib-python/tree/master/examples/logging)
[Examples of setting parameters](https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/parameters/basicparam.py)

**Reply by hssjung** [reply]
Thanks. I got the distance from the anchor for your help.
